src/terra-dev-site/about/ComponentStandards.b.about.mdx

```diff
@@ -47,19 +49,19 @@ Terra components are built following the mobile-first approach and utilize the f
 
 Components should:
 * Support gestures when possible (swiping / pinching)
-* Not use hover events
+* Not use hover events without an alternative method of accessing the same information
 * Avoid small tap target sizes
 
 ## Accessibility (A11y)
 
-We should strive to make components accessible. Use the following guides to help build accessible components.
+Terra components are accessible. Use the following guides to help build accessible components.
 * [WCAG 2.0](https://www.w3.org/TR/WCAG20/)
 * [Section 508](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/text-of-the-standards-and-guidelines)
 * [Color Contrast Checker](http://webaim.org/resources/contrastchecker/)
 
-Components should support keyboard navigation.
+Components support keyboard navigation.
 
-All functionality that is available by mouse or touch should also be available by keyboard. Keyboard operation is possible by moving to ('focusing') interactive elements and activating them using keystrokes.
+All functionality that is available by mouse or touch must also be available by keyboard. Keyboard operation is possible by moving to ('focusing') interactive elements and activating them using keystrokes.
```

Comment:
Is the purpose of this doc to describe how Terra components _are_ or to describe to consumers how Terra components _must be_? We either need to more/less 'musts' depending on what we're going for.

Reply:
The problem is this doc pulls double duty for both describing terra's features and informing contributors of the standards they must meet. We probably need fewer musts.